test/JDBC/upgrade/latest/schedule

```diff
@@ -102,6 +102,7 @@ BABEL-2917
 BABEL-2955
 BABEL-3358
 BABEL-3747
+BABEL-3781
```

Comment:
We should use PR template for commit message. Also, latest schedule file is for Major version upgrade testing. To test it for minor version upgrade, add it to 14.6 schedule file